ui/admin/app/components/scope-picker/index.hbs

```diff
@@ -0,0 +1,98 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<Hds::Dropdown
+  @enableCollisionDetection={{true}}
+  @matchToggleWidth={{true}}
+  class='hds-side-nav-hide-when-minimized scope-picker'
+  as |D|
+>
+  {{#let this.currentScope as |scope|}}
+    <D.ToggleButton
+      @text={{scope.name}}
+      @icon={{scope.icon}}
+      class='scope-picker__toggle-button'
```

Comment:
Looks like this class is never used?

Reply:
great catch!!

Reply:
Aren't you setting the width to 100% with this?

Reply:
oh yeah ur right, when i did the initial search I missed it 🤦